happi/backends/mongo_db.py

```diff
@@ -21,35 +21,51 @@ class MongoBackend(_Backend):
 
     Parameters
     ----------
-    host : str, optional
+    host : str
         Hostname for MongoDB.
-    user : str, optional
+    user : str
         Username for MongoDB instance.
-    pw : str, optional
+    pw : str
         Password for given username.
-    host : str, optional
-        Host of the MongoDB instance.
-    db : str, optional
+    db : str
         Database name within the MongoDB instance.
+    collection : str
+        MongoDB colleciton name
     timeout : float, optional
         Time to wait for connection attempt.
-    cfg_path : str, optional
-        Path to the happi config.
+    port : str, optional
```

Comment:
Port _should_ be int, so it's a little odd to assume that someone will pass in a string (even if it is the typical source from a config file):

```suggestion
    port : int or str, optional
```